src/Microsoft.Kiota.Cli.Commons/IO/FormatterType.cs

```diff
@@ -6,9 +6,13 @@
 public enum FormatterType
 {
     /// <summary>
-    /// JSON format
+    /// Raw JSON format.
     /// </summary>
-    JSON,
+    RAW_JSON,
+    /// <summary>
+    /// Raw JSON format.
+    /// </summary>
+    PRETTY_JSON,

```

Comment:
I'll ask about this in the Admin Experiences chat.

Reply:
Just to confirm, what we mean here is dropping the `JSON` value to use `TEXT` since its essentially unformatted content? Or simply renaming `RAW_JSON` to `JSON`?

Reply:
@andrueastman, I think it's renaming `PRETTY_JSON` to `JSON`. I'm unsure at the moment whether to drop `RAW_JSON` and use text. I haven't figured out how to hide some enum variants in System.CommandLine. Maybe I'll have to change the option to a free text with generated content. JSON is only valid if an endpoint supports `JSON` output so we can't simply replace it with `TEXT` (especially because we post-process it to add indents).